### ISSUE TITLE
/Trylevelevo failing to work sometimes.

### ISSUE THREAD

**Issue by Menno165** [NebulaMC-GG/Support#1641]
Sometimes /trylevelevo does not work, even if the right requirements are met.

Will post more information possible if obtained.

**Comment by Thenatural91** [comment]
I had an issue with multiple pokemon evolving. My rookidee didn't evolve into Corvisquire, my Stufful into Bewear, and Squirtle into Wartortle. My Rookidee and Stufful were my first ones. They reached the level requirements to evolve, but didn't I tried /trylevelevo and nothing. Their sprites would flash but no animation or evolution. I boxed them logged off and restarted, tried again, but nothing. I boxed them and left them there while I trained other Pokemon for an hour and a half. I took a break for dinner, closed all the way out of the technic launcher. When I came back, I took out Stufful and Rookidee to try again as well as my recently hatched Squirtle. All three still wouldn't evolve. Stufful and Squirtle gave me the "Pokemon evolved into Pokemon" upon level up
![2022-07-12_17 41 45](https://user-images.githubusercontent.com/31575870/178608771-9676d6a5-3c78-4f24-b43b-2a5ac0a031ae.png)
.
